### PR TITLE
map convert

### PR DESCRIPTION
**Pull request by Kuro2709**
modify map conversion function

### CODE REVIEW

**Review by kingsawpdx — APPROVED** [review]
Looks good to me!

File: src/DataStructures.ts [hunk 1]
```diff
@@ -17,6 +17,9 @@ export interface VFE {
 // Navigation map: a birdseye view of the various hotspots within a single 360-environment
 export interface NavMap {
   src: string;
+  center: { x: number; y: number };
+  rotation: string;
```
Comment (mnxn) [inline src/DataStructures.ts:21]: As mentioned in the meeting, the `rotation` field should have a `number` type.

**Review by mnxn — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 2]
```diff
@@ -181,8 +182,20 @@ function App() {
   };
 
   const map: NavMap = {
-    src: "map src",
-    hotspots: [],
+    src: mapImage,
+    center: { x: 450, y: 800 },
+    rotation: parseInt("0deg"),
```
Comment (mnxn) [inline src/App.tsx:187]: This line should just be `rotation: 0`.

**Review by mnxn — CHANGES_REQUESTED** [review]

**Review by mnxn — APPROVED** [review]
Looks good.

**Pull request merged**